### ISSUE TITLE
Manifest version should be mutually exclusive with csproj version

### ISSUE THREAD

**Issue by philpax** [goatcorp/Plogon#16]
Unclear what the current state of affairs is, but https://discord.com/channels/581875019861328007/929533532185956352/1009523451490484325 suggests that the manifest takes precedence over the csproj.

This shouldn't be the case - we ratified mutual exclusion here https://github.com/goatcorp/DIPs/pull/44 to ensure that we never have two versions that are out of sync.

**Comment by wolfcomp** [comment]
That is correct due to how manifest version is read and sent to the build and with how dotnet handles the build args it's unable to be written in any other way see [BuildProcessor](https://github.com/goatcorp/Plogon/blob/master/Plogon/BuildProcessor.cs#L506) and [entrypoint](https://github.com/goatcorp/Plogon/blob/master/Plogon/static/entrypoint.sh#L7)

**Comment by philpax** [comment]
I mean, I don't see why it couldn't accept both versions and error out during the build if both are present, instead of just one.
